edenai_apis/apis/tabscanner/tabscanner_api.py

```
@@ -241,3 +252,144 @@ def ocr__financial_parser(
+        async with async_client(OCR_TIMEOUT) as client:
+            response = await client.get(self.url + "result/" + token, headers=headers)
+        response_json = response.json()
+        if response_json["status"] == "pending" and retry <= 5:
+            await asyncio.sleep(1)
+            return await self._aget_response(token, retry + 1)
+        return response_json, response.status_code
+
+    async def ocr__afinancial_parser(
+        self,
+        file: str,
+        language: str,
```

Ruff (0.14.11)

Unused method argument: `language`

(ARG002)